R/get-results.r

```diff
@@ -645,6 +649,15 @@ get_compfit <- function(report.file, name) {
 #' @return A dataframe
 make_df <- function(list_name, list_df) {
   list_df_comp <- lapply(list_df, function(x) x[[list_name]])
+  # set anything not a dataframe (the NA values) as Null
+  list_df_comp <- lapply(list_df_comp, function (x) {
+      if(!is.data.frame(x)) {
+        x <- NULL
```

Comment:
Line 329-332 uses NA rather than NULL

Reply:
this was intentional; NULL is being used here in order to completely get rid of NA elements, which causes downstream code to fail.

It is possible that lines 329-332 could use NULL instead of NA, but I wasn't sure if this would cause problems in some situations, so left it as-is.

Reply:
What happens if all elements here are NULL, would and you use `purrr::compact(list_df_comp)`? Would the final list have an NA value? Just trying to make sure that the output of `get_results_mod` is the same whether or not the function exists early.

Reply:
Good point, thanks for pointing this out! I'm adding a test for this and will push up changes that accounts for this situation.